hack/cnbi-import.yaml

```diff
@@ -193,3 +194,173 @@ spec:
             EOM
 
             oc patch imagestream $(context.pipelineRun.name) -n $(context.pipelineRun.namespace) --type="merge" --patch-file patch-file.yaml
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: byon-validate-jupyterhub-image
+  labels:
+    app.kubernetes.io/created-by: byon
+spec:
+  params:
+    - name: url
+      description: Container image repository url
+      type: string
+  workspaces:
+    - name: data
+  steps:
+    - name: get-package-versions
+      image: $(params.url)
```

Comment:
Does being under 'hack' means it's a temporary (in a POC way) ?

Do we trust our params enough to directly run the image ?

Reply:
it is a way to indicate the POC'ness of the work (at least for me it is)

Reply:
> Does being under 'hack' means it's a temporary (in a POC way) ?

It is a consequence of the PoC-ness indeed. Also, I believe we have not fully decided where to store and maintain the pipelines. https://github.com/thoth-station/opendatahub-cnbi/issues/10

The pipeline manifests under `hack/` have been taken from the helm-charts repo and adapted.

> Do we trust our params enough to directly run the image ?

There should be validation...